.eslintignore

```diff
@@ -1,12 +1,16 @@
+packages/@react-aria/i18n/server
 packages/@spectrum-icons/color/**
 packages/@spectrum-icons/ui/**
 packages/@spectrum-icons/workflow/**
 packages/@spectrum-icons/illustrations/**
 packages/@spectrum-icons/express/**
 node_modules
 packages/*/*/dist
+packages/*/*/i18n
 packages/react-aria/dist
+packages/react-aria/i18n
 packages/react-aria-components/dist
+packages/react-aria-components/i18n
```

Comment:
not sure this was intentional or not?

Reply:
It was, they need to be added to our clean function as well.
I think they came from build, possibly in another branch. Either way, they need to be ignored and/or removed by clean_all

Reply:
ah I misunderstood, I thought this was ignoring the i18n files like `useFilter`, makes sense